packages/apputils/test/mainareawidget.spec.ts

```diff
@@ -23,6 +23,25 @@ describe('@jupyterlab/apputils', () => {
         expect(widget.hasClass('jp-MainAreaWidget')).toBe(true);
         expect(widget.toolbar).toBe(toolbar);
       });
+
+      it('toolbar should have an aria label of notebook actions and a role of navigation', () => {
+        const content = new Widget();
+        const toolbar = new Toolbar();
+        const widget = new MainAreaWidget({ content, toolbar });
+        expect(widget.toolbar.node.getAttribute('aria-label')).toEqual(
+          'notebook actions'
+        );
+        expect(widget.toolbar.node.getAttribute('role')).toEqual('navigation');
+      });
+
+      it('content should have an aria-label of notebook content and a role of region', () => {
+        const content = new Widget();
+        const widget = new MainAreaWidget({ content });
+        expect(widget.content.node.getAttribute('aria-label')).toEqual(
+          'notebook content'
```

Comment:
This test demonstrated incorrect behaviour (not every main area widget is a notebook, for example a text editor is not a notebook, csv viewer is not a notebook etc), see https://github.com/jupyterlab/jupyterlab/issues/13045. I am not sure what the default for a main area widget should be but my expectation is that it would not be "notebook". Also, is the toolbar really `navigation`? Shouldn't it be a [toolbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role)?

Reply:
I can change the navigation role to toolbar. As the notebook content label still has an outstanding issue, I could either remove the test or leave it to be updated when that issue is worked on?

Reply:
Let's keep the test, but:
- mark it with [`test.failing`](https://jestjs.io/docs/api#testfailingname-fn-timeout) as xfail for now
- add a comment on the test pointing to issue
- once this PR is merged, comment on the issue that there is now a test.

Reply:
Changes made.